index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Tribute Page</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+
+  <body>
+    <main id="main">
+      <h1 id="title">Percy Lavon Julian</h1>
+      <div id="img-div">
+        <img
+          id="image"
+          src="https://www.sciencehistory.org/sites/default/files/styles/rte_full_width/public/rte/percy_julian_lab.jpg?itok=kWOzYHzF"
+          alt="Percy"
+        />
+        <p id="img-caption">
+          Percy Julian working in a DePauw University lab, ca. 1930. 📷 From the
+          collection of the Historical Society of Oak Park and River Forest
+        </p>
+      </div>
```

Comment:
instead of using div, which is not semantic, try adding the img and p tags inside a figure tag. This will group them together and would be better for your code.